### PR TITLE
update banner copy and cta link for moment

### PR DESCRIPTION
**Pull request by jlieb10**
## What does this change?
Copy change on the moment banner, updating the link

also cross browser tested the banner

## Screenshots
moment banner update 1
moment banner update 2

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/contributions-moment-banner-environment.js [hunk 1]
```diff
@@ -6,21 +6,21 @@ import { canShowBannerSync } from 'common/modules/commercial/contributions-utili
...
 
 const geolocation = geolocationGetSync();
-const campaignId = 'environment_campaign';
+const campaignId = 'climate_pledge_2019';
 
 const defaultLeadSentence =
     'The climate emergency is the defining issue of our times. ';
 const defaultCopy =
-    'This is The Guardian’s pledge: we will be truthful, resolute and undeterred in pursuing our journalism on the environment. We will give global heating, wildlife extinction and pollution the urgent attention they demand. Our independence means we can interrogate inaction by those in power. It means our reporting will always be grounded in scientific facts, never in commercial or political interests. Support from our readers makes this work possible.';
+    'This is the Guardian’s pledge: we will be truthful, resolute and undeterred in pursuing Guardian journalism on the environment. We will give global heating, wildlife extinction and pollution the urgent attention they demand. Our independence means we can interrogate inaction by those in power. It means Guardian reporting will always be driven by scientific facts, never by commercial or political interests. Support from our readers makes this work possible.';
```
Comment (tjsilver) [inline static/src/javascripts/projects/common/modules/experiments/tests/contributions-moment-banner-environment.js]: Just curious - but isn't this inconsistent with the way we refer to "The Guardian" in other places...?

**Review by tjsilver — APPROVED** [review]
Great work!

**Review by tomrf1 — APPROVED** [review]
👍

File: static/src/javascripts/projects/common/modules/experiments/tests/contributions-moment-banner-environment.js (hunk 1 above)
Comment (jlieb10) [reply static/src/javascripts/projects/common/modules/experiments/tests/contributions-moment-banner-environment.js]: I've only seen it as 'The Guardian' as well... @ionamckendrick is this an editorial style change?

**Pull request merged**